core/shared/src/main/scala/scalaz/zio/duration/Duration.scala

```diff
@@ -0,0 +1,73 @@
+package scalaz.zio.duration
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.{ Duration => ScalaDuration, FiniteDuration => ScalaFiniteDuration }
+import scala.math.Ordered
+
+sealed trait Duration extends Ordered[Duration] {
```

Comment:
👍

Reply:
Sure. I also want to add Duration tests in general. I should get it done this week. And I'm already looking for new things to work on in ZIO!

Reply:
@jdegoes actually serialization works either way as case classes extend Serializable with Product. So now I'm wondering if adding Serializable with Product to the trait isn't meaningless?